exercises/practice/protein-translation/.meta/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "authors": [],
```

Comment:
issue: missing username

Reply:
Should this be GitHub username or Exercism username?   vlang README.md says `Author's GitHub username`  but I haven't received Exercism reputation points for any pull requests.

(I created my **keiraville** Exercism account first, then found out the corresponding GitHub username was already taken, hence I'm using @keiravillekode on GitHub.)

Reply:
I believe it should be Github. Have you linked your exercism account with your github? that's necessary to receive the reputation points

Reply:
https://exercism.org/settings shows

> Github (Handle)
> keiravillekode
> 

https://exercism.org/settings/integrations shows

> GitHub
> Your Exercism and GitHub accounts are linked. 
>

Reply:
I have updated `config.json` with my GitHub username, it matches my earlier contributions.

Reply:
> https://exercism.org/settings shows
> 
> > Github (Handle)
> > keiravillekode
> 
> https://exercism.org/settings/integrations shows
> 
> > GitHub
> > Your Exercism and GitHub accounts are linked.

oh weird... have you asked about that on the forum? sorry I can't help more than that!

Reply:
I haven't. Thanks.

https://exercism.org/docs/building/tracks/concept-exercises

> authors: The GitHub username(s) of the exercise's author(s) (required)

> contributors: The GitHub username(s) of the exercise's contributor(s) (optional)